tests/playwright/assets/breakpoints.ts

```diff
@@ -33,6 +33,29 @@ export default class {
 		return [ 'mobile', 'tablet', 'desktop' ];
 	}
 
+	async saveOrUpdate( editor: EditorPage, toReload = false ) {
+		const hasTopBar: boolean = await editor.hasTopBar();
+		if ( hasTopBar ) {
+			const saveButton = "//button[text()='Save Changes']";
+			if ( await this.page.locator( saveButton ).isEnabled() ) {
+				await this.page.locator( saveButton ).click();
+			} else {
+				await this.page.evaluate( ( selector ) => {
+					const button: HTMLElement = document.evaluate( selector, document, null, XPathResult.FIRST_ORDERED_NODE_TYPE ).singleNodeValue as HTMLElement;
+					button.click();
+				}, saveButton );
+			}
+			if ( toReload ) {
+				const reloadButton = this.page.locator( 'button', { hasText: 'Reload Now' } );
+				await reloadButton.waitFor();
+				await reloadButton.click();
+			}
+		} else {
+			await this.page.locator( 'text=Update' ).click();
+			await this.page.waitForSelector( '#elementor-toast' );
```

Comment:
```suggestion
			await this.page.locator( '#elementor-toast' ).waitFor();
```

See the recommendations from this file: https://elementor.atlassian.net/wiki/spaces/RDDEP/pages/edit-v2/835617108.